forms-flow-theme/scss/_forms.scss

```diff
@@ -402,4 +402,70 @@ select option:hover {
 
 .valiation-astrisk {
   color: var(--ff-base-600);
+}
+
+.dropdown {
```

Comment:
is it possible to give the styling to general select dropdown so that it would look similar to formio dropdown as well. if it is complex, we can think about it later